manifests/init.pp

```diff
@@ -386,7 +387,7 @@
   Integer[0] $list_max_ziplist_value                             = 64,
   Stdlib::Absolutepath $log_dir                                  = $redis::params::log_dir,
   Stdlib::Filemode $log_dir_mode                                 = $redis::params::log_dir_mode,
-  String $log_file                                               = 'redis.log',
+  Variant[Boolean[false], String[1]] $log_file                   = 'redis.log',
```

Comment:
could you please add a unit test that sets this to false?

Reply:
Grrr. Done.